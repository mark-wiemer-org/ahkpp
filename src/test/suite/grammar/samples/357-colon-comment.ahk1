; [Issue #357](https://github.com/mark-wiemer/vscode-autohotkey-plus-plus/issues/357)
; Test::Comment
    ; Test::Comment
