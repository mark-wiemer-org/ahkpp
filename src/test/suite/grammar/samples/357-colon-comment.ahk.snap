>; [Issue #357](https://github.com/mark-wiemer/vscode-autohotkey-plus-plus/issues/357)
#^ source.ahk comment.line.semicolon.ahk punctuation.definition.comment.ahk
# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk comment.line.semicolon.ahk
>; Test::Comment
#^ source.ahk comment.line.semicolon.ahk punctuation.definition.comment.ahk
# ^^^^^^^^^^^^^^ source.ahk comment.line.semicolon.ahk
>    ; Test::Comment
#^^^^ source.ahk comment.line.semicolon.ahk
#    ^ source.ahk comment.line.semicolon.ahk punctuation.definition.comment.ahk
#     ^^^^^^^^^^^^^^ source.ahk comment.line.semicolon.ahk
>